docs/.vuepress/styles/prism-theme.min.css

```diff
@@ -18,108 +29,116 @@ pre[class*='language-'] {
+pre[class*="language-"]::mozselection,
+code[class*="language-"]::mozselection {
   text-shadow: none;
-  background: #011627;
-}
-code[class*='language-'] ::selection,
-code[class*='language-']::selection,
-pre[class*='language-'] ::selection,
-pre[class*='language-']::selection {
-  text-shadow: none;
-  background: #011627;
+  background: var(--blue-100);
 }
 @media print {
-  code[class*='language-'],
-  pre[class*='language-'] {
+  pre[class*="language-"],
+  code[class*="language-"] {
     text-shadow: none;
   }
 }
-pre {
+pre[class*="language-"] {
   padding: 1em;
-  margin: 0.5em 0;
+  margin: .5em 0;
   overflow: auto;
+  background: var(--grey-98);
+}
+pre[class*="language-"] code,
+:not(pre) > code[class*="language-"] {
+  padding: .1em .3em;
+  border-radius: .3em;
+  color: var(--blue-600) !important;
+  background: var(--grey-92);
+}
+/*********************************************************
+* Tokens
+*/
+.namespace {
+  opacity: .7;
 }
-:not(pre) > code,
-pre {
-  background: #011627;
-}
-:not(pre) > code {
-  padding: 0.1em;
-  border-radius: 0.3em;
-  white-space: normal;
-}
-.token.cdata,
 .token.comment,
-.token.prolog {
-  color: #637777;
-  font-style: italic;
+.token.prolog,
+.token.doctype,
+.token.cdata {
+  color: var(--black-70);
 }
 .token.punctuation {
-  color: #c792ea;
-}
-.namespace {
-  color: #b2ccd6;
-}
-.token.deleted {
-  color: rgba(239, 83, 80, 0.56);
-  font-style: italic;
-}
-.token.property,
-.token.symbol {
-  color: #80cbc4;
-}
-.token.keyword,
-.token.operator,
-.token.tag {
-  color: var(--blue-light-01);
-}
-.token.boolean {
-  color: #ff5874;
+  color: var(--black-70);
 }
-.token.number {
-  color: #f78c6c;
-}
-.token.builtin,
-.token.char,
+.token.tag,
+.token.boolean,
+.token.number,
 .token.constant,
-.token.function {
-  color: #82aaff;
-}
-.token.doctype,
-.token.selector {
-  color: #c792ea;
-  font-style: italic;
+.token.symbol,
+.token.deleted {
+  color: var(--blue-600);
 }
+.token.selector,
 .token.attr-name,
+.token.string,
+.token.char,
+.token.builtin,
 .token.inserted {
-  color: #addb67;
-  font-style: italic;
+  color: var(--green-500);
 }
+.token.operator,
+.token.entity,
+.token.url,
 .language-css .token.string,
 .style .token.string,
-.token.entity,
-.token.string,
-.token.url {
-  color: #addb67;
+.token.class-name {
+  color: var(--green-500);
+  background: transparent;
 }
 .token.atrule,
 .token.attr-value,
-.token.class-name {
-  color: #ffcb8b;
+.token.keyword {
+  color: var(--blue-400);
+}
+.token.property,
+.token.function {
+  color: var(--red-600);
 }
-.token.important,
 .token.regex,
+.token.important,
 .token.variable {
-  color: #d6deeb;
+  color: var(--green-500);
 }
-.token.bold,
-.token.important {
-  font-weight: 700;
+.token.important,
+.token.bold {
+  font-weight: bold;
 }
 .token.italic {
   font-style: italic;
 }
+.token.entity {
+  cursor: help;
+}
+/*********************************************************
+* Line highlighting
```

Comment:
Line highlighting is fugly
```
```bash{1}
```
![image](https://user-images.githubusercontent.com/5770711/85589643-b7a58a00-b611-11ea-9b02-6d761e18f36c.png)

Reply:
oh very weird, I actually themed that!
![image](https://user-images.githubusercontent.com/5941111/85590394-38a45780-b5f9-11ea-842c-eb9b2c807c39.png)

Reply:
fixed numbers and highlighting
![image](https://user-images.githubusercontent.com/5941111/85592193-cc2a5800-b5fa-11ea-9c8b-b70af160265b.png)